app/components/primary-button.tsx

```diff
@@ -0,0 +1,12 @@
+import { Button } from "@/components/ui/button";
+
+interface IButtonInfo {
```

Comment:
コンポーネントの引数 (props) のインターフェースは、コンポーネント名 + Props と命名されるのが一般的ですねー
```suggestion
interface PrimaryButtonProps {
```
interface を使っているのは good です